README.md

```diff
@@ -29,7 +29,7 @@ This GTK theme is based on the [Colloid](https://github.com/vinceliuice/Colloid-
 
 ### Installation
 
-1. Download and extract the theme zip from [releases](https://github.com/catppuccin/gtk/releases/) or you can install the theme from the [AUR](#for-arch-linux-users).
+1. Download and extract the theme zip from [releases](https://github.com/catppuccin/gtk/releases/).
```

Comment:
why remove the AUR?

Reply:
Hey there @npv12,

I didn’t think the AUR belonged in the line due to the fact that the AUR packages handle the path moving stuff automatically and there is a section for the AUR under anyways.

Reply:
Makes sense. Thanks for the contribution. Merging it